api/src/repositories/security-repository.ts

```diff
@@ -160,6 +160,26 @@ export class SecurityRepository extends BaseRepository {
     await this.connection.sql(sqlStatement);
   }
 
+  /**
+   * Delete persecution rules for a given artifact uuid
```

Comment:
Debated if this was a valid comment or not, certainly subjective, so feel free to ignore: my instinct would be to add "if any exist" here, to help explain why we don't have any error handling. 

Something like: "Deletes all persecution and harm rules, if any exist, for a given artifact uuid"

Reply:
updated